kvm.go

```diff
@@ -134,7 +142,7 @@ func (d *Driver) GetCreateFlags() []mcnflag.Flag {
 		mcnflag.StringFlag{
 			Name:  "kvm-io-mode",
 			Usage: "Disk IO mode: threads, native",
-			Value: "threads",
+			Value: "native",
```

Comment:
OK, reverted the change to switch the default IO mode from threads to native and kept it on threads.